### DISCUSSION TITLE
Email - Template, Translation, trigger

### DISCUSSION THREAD

**Discussion by MrCasquette** [directus/directus#14249] [Archive]
Hi I wanted to know how to translate Email from directus, or how to overwrite an existing template in the self hosted version. If it's possible of course. Thank you for your time :)

**Comment by Pachat** [comment]
ATM, on a server install, we have for Sharing a link by mail
Subject: `xx User has shared an item with you`
and Body:
```
Hello!

xx User has invited you to view an item in test.

[Open](http://admin/shared/4251110c-f5c9-462a-8f7c-a2c2d1a06540)

```
So, besides translation, on a new installation on a VPS server _(no domain installed yet: Directus is working fine with the IP)_

- Obviously, the link does not work

- The mail is sent from `Directus <no-reply@directus.io>`

- Has on top `Directus Logo` 

- and at bottom `Sent by the team at Directus — Manage Emails`

Looks like that some customization would be appreciated.

**Reply by MrCasquette** [reply]
I decided to find a solution anyway. I get on my VPS to the directus folder. I f you go into this url : /node_modules/directus/dist/services/mail/templates you have the 3 email templates. Then just had to overwrite with what you want depending on the template context and your language obviously. Hope it helps
